cmd/k8s-operator/proxygroup.go

```diff
@@ -520,12 +520,16 @@ func (r *ProxyGroupReconciler) ensureConfigSecretsCreated(ctx context.Context, p
 		}
 
 		if existingCfgSecret != nil {
-			logger.Debugf("patching the existing ProxyGroup config Secret %s", cfgSecret.Name)
-			if err := r.Patch(ctx, cfgSecret, client.MergeFrom(existingCfgSecret)); err != nil {
-				return "", err
+			if apiequality.Semantic.DeepEqual(existingCfgSecret, cfgSecret) {
+				logger.Debugf("Secret unchanged, skipping update for existing ProxyGroup config Secret %s", cfgSecret.Name)
```

Comment:
Do we need this line? I think it does not add extra info (it should already be possible to observe in which cases updates did not happen due to the absence of the below 'Updating..' line in between a 'reconcile started..', 'reconcile finished..' and we do already log loads in debug mode

Reply:
Yeah that's fair 👍  removed it